### DISCUSSION TITLE
Oriented bounding boxes orientation?

### DISCUSSION THREAD

**Discussion by ToVi11** [isl-org/Open3D#5917] [Q&A]
Hi,

I have an oriented bounding box and I would like to do two things:
1. To visualize the normal vector from the center of the bounding box. 
2. To rotate the  points inside a bounding box so that I get perpendicular projection to YZ plane

I now that these both can be achieved by the bounding box properties center and R (rotation matrix) but I cannot figure how. Could someone help me?

**Comment by benjaminum** [comment]
Hi,
not sure what the normal is in your example.
For 2., the rotation matrix of the oriented bounding box can be used as the new coordinate frame.
You can use $R$ to transform the points $x^{'} = Rx$
